src/Nethermind.Verkle/Proofs/Quotient.cs

```diff
@@ -31,6 +31,28 @@ public static FrE[] ComputeQuotientInsideDomain(PreComputedWeights preComp, Lagr
         return q;
     }
 
+    public static void ComputeQuotientInsideDomain(PreComputedWeights preComp, LagrangeBasis f, byte index, Span<FrE> quotient)
```

Comment:
@tanishqjasoria, maybe you can make the overloaded method at L9 to call this one now? To avoid repeating logic, since this new one is more general.

Reply:
ah i meant to delete that function - its not used anywhere now